localstripe/localstripe-v3.js

```diff
@@ -80,20 +80,49 @@ function openModal(text, confirmText, cancelText) {
 }
 
 class Element {
-  constructor() {
+  constructor(elements) {
     this.listeners = {};
+    // Element needs a reference to the object that created it, in order to
+    // thoroughly destroy() itself.
+    this._elements = elements;
```

Comment:
- Much better with a leading `_` for these internal properties :+1: 
- What about `parent` instead of `elements`? It's a bit clearer to me, but this is personal taste. Take the proposal or don't.

Reply:
I think `stripeElements` is clearer than either one. The naming is a little tricky because we are working with both DOM elements and Stripe elements, and the DOM has its own concept of "parent".

Reply:
`this._stripeElements` sound perfect to me :+1: